public/employers/groupon.yml

```diff
@@ -0,0 +1,43 @@
+name: Groupon, Inc.
+shortName: Groupon
+ticker: GRPN
+wiki: Groupon
+image: groupon.svg
+officialWebsite: https://press.groupon.com/about-groupon/
+industries:
+  - eCommerce
+location:
+  city: Chicago
+  state: IL
+  country: us
+  international: true
+  wiki: Chicago
+employeesBefore:
+  type: approximately
+  upperBound: 7000
+  year: 2020
+summary: Groupon implemented a number of useful cost-cutting measures, such as reducing marketing initiatives, but also resorted to furloughs while not reducing cash compensation for executive board members.
+citations:
+  - summary: Groupon let go of combined total of 2,800 workers&mdash;40% of their workforce&mdash;through layoffs and furloughs.
+    positivity: -1
+    type: publication
+    sources:
+      - source: The Motley Fool
+        link: https://web.archive.org/web/20200423022246/https://www.fool.com/investing/2020/04/15/groupon-to-lay-off-or-furlough-2800-workers-initia.aspx
+        date: 2020-04-15T13:09:00Z
+  - summary: Groupon implemented a hiring freeze, reduced marketing initiatives, and other cost-cutting measures.
+    positivity: 1
+    type: publication
+    sources:
+      - source: The Motley Fool
+        link: https://web.archive.org/web/20200423022246/https://www.fool.com/investing/2020/04/15/groupon-to-lay-off-or-furlough-2800-workers-initia.aspx
+        date: 2020-04-15T13:09:00Z
+      - source: CNBC
+        link: https://web.archive.org/web/20200426010552/https://www.cnbc.com/2020/03/13/workforce-wire-coronavirus-heres-what-every-major-company-is-doing-about-the-pandemic.html
```

Comment:
Date for this is MAR 13 20205:17 PM EDT

Reply:
The initial post is, but they're updating this page constantly. As a result I don't know when the information was added for this particular company, so I'm not specifying a particular date for this one.